docs/source/upcoming_release_notes/1154-add_TIXEL_Manipulaotr_Motors_to_Magenetic_Bottle.rst

```diff
@@ -0,0 +1,30 @@
+1154 DOC: add TIXEL Manipulaotr Motors to Magenetic Bottle
```

Comment:
minor funny typos here but I don't think this ends up in the final release docs so it's ok